apollo-federation/src/query_graph/path_tree.rs

```diff
@@ -718,10 +717,12 @@ mod tests {
 
         let normalized_operation = normalize_operation(
             operation,
-            Default::default(),
+            &Default::default(),
             &schema,
             &Default::default(),
-            &never_cancel,
+            &||
```

Comment:
Can we keep `never_cancel` here?

Reply:
https://github.com/apollographql/router/pull/6960/commits/8643ec5b03071127634cd238ed797d63856ad595